rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -318,7 +330,7 @@ public static LockWhitelist deserializeLockWhitelist(byte[] data, NetworkParamet
 
         for (ReleaseRequestQueue.Entry entry : entries) {
             bytes[n++] = RLP.encodeElement(entry.getDestination().getHash160());
-            bytes[n++] = RLP.encodeBigInteger(BigInteger.valueOf(entry.getAmount().getValue()));
+            bytes[n++] = serializeCoin(entry.getAmount());
```

Comment:
It sucks we can't use `deserializeCoin` because of how our `RLP` library is coded. I'm undoing this for now, but I think that code reuse is always a good idea.